CHANGELOG.md

```diff
@@ -19,11 +19,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Support for OpenGL ES 2.0
 - Escape sequence to set underline color (`CSI 58 : 2 : Ps : Ps : Ps m`/`CSI 58 : 5 : Ps m`)
 - Escape sequence to reset underline color (`CSI 59 m`)
+- Support for inline input method
```

Comment:
Do we want to mention that the caveat is that on x11 only off-the-spot preedit area (no over-the-spot or on-the-spot) is supported due to library limitations (xcb not available)?

Reply:
I'm not entirely sure. It's kind of common and I've seen other clients with such issue.

@chrisduerr what do you think?

Reply:
I have absolutely no clue what these different IME modes are, but chances are if we just say "support XYZ" and Toby's version of XYZ doesn't work he'll come crying on the issue tracker. So we should set the expectations so people are aware of which setups should and shouldn't work now?

Reply:
Something like `IME is now stuck at the bottom of the window due to libX11 limitations` in CHANGED?

Reply:
`IME is only available at the bottom of the window (off-the-spot preedit area) due to libX11 limitations for x11`

Reply:
I don't know, I kind of like the one I have in CHANGELOG right now, since it's more about changed.

Reply:
It's also about changed, but at least it needs to mention the unexpected, better than let users dig in to figure out why it didn't work as they expected.